### PR TITLE
Excel] (custom functions) remove 'UI-less' terminology

### PR DESCRIPTION
**Pull request by Rick-Kirkham**

### CODE REVIEW

**Review by alison-mk — APPROVED** [review]
Looks good! Thanks for making these changes. Left comments about a couple very minor changes, and one question.

File: docs/toc.yml [hunk 1]
```diff
@@ -560,12 +560,12 @@ items:
     - name: Return a dynamic array
       href: excel/custom-functions-dynamic-arrays.md
       displayName: Excel, Custom Functions
-    - name: UI-less custom functions
+    - name: Custom functions without a shared runtime
```
[the remark this answers is not in the document]
Comment (Rick-Kirkham) [reply docs/toc.yml:563]: I'd say yes, since it is about debugging only custom functions without a shared runtime. But I'll leave that to you.

**Review by davidchesnut — CHANGES_REQUESTED** [review]

**Review by davidchesnut — COMMENTED** [review]
I really like the phrasing of "custom functions without a shared runtime." Perfect rename!

File: docs/resources/resources-glossary.md [hunk 2]
```diff
@@ -131,9 +131,9 @@ A **tutorial** is a teaching aid designed to help people learn to use a product
 
 See also: [quick start](#quick-start).
 
-## UI-less custom function
+## custom functions-only add-in
 
-**UI-less custom functions** run in the custom functions runtime. They have no UI and cannot interact with Office.js APIs.
+An add-in that contains a custom function, but no UI such as a task pane. The custom functions in this kind of add-in run in a JavaScript-only runtime. A custom function that does include a UI can use either a shared runtime or a combination of a JavaScript-only runtime and an HTML-supporting runtime.
```
[the remark this answers is not in the document]
Comment (Rick-Kirkham) [reply docs/resources/resources-glossary.md:136]:
```suggestion
An add-in that contains a custom function, but no UI such as a task pane. The custom functions in this kind of add-in run in a JavaScript-only runtime. A custom function that does include a UI can use either a shared runtime or a combination of a JavaScript-only runtime and an HTML-supporting runtime. We recommend that if you have a UI, you use a shared runtime. 
```

**Pull request merged**